### ISSUE TITLE
If Jarvis fails, he won't stop failing.

### ISSUE THREAD

**Issue by cherche** [JaredCubilla/jarvis#28]
If Jarvis is given a string he can't respond to and must perform a Google Search, being given a string that Jarvis _can_ respond to properly afterward results in two outputs: the Google Search response, and the correct response.
![screen shot 2015-12-24 at 2 16 42 pm](https://cloud.githubusercontent.com/assets/12107277/11997449/8e159690-aa49-11e5-8b99-2f7a28175f38.png)
![screen shot 2015-12-24 at 2 17 03 pm](https://cloud.githubusercontent.com/assets/12107277/11997450/8e22fef2-aa49-11e5-856e-875937cc6843.png)

**Comment by JaredCubilla** [comment]
Yeah, this probably has to do with the how the movie data is being retrieved. I think it's asynchronous while pretty much everything is synchronous. I should probably have the data retrieval trigger some sort of event that will notify the user to wait or something, until the data comes back fine. Thanks!
